<div id="contents" style="margin-top:7%;">
		<div id="about" class="body">
			<h1>Softwares</h1>
			<h4><a href="http://discovery.informatics.iupui.edu/SPINNER/">SPINNER</a></h4>
			
					<p>
						SPINNER stands for Seeded Protein Interaction Network Neighborhood Expansion and Ranking Tool. It is an automated software tool, which can rank and compare genes or proteins from constructed phenotype-specific biomolecular interaction networks. Given the user input of a list of phenotype-specific genes, our tool can query the STRING protein-protein interaction database automatically to retrieve protein-protein interactions among the input genes with user-specified network expansion levels to construct a phenotype-specific network. All the sub-networks are ranked and evaluated statistically to obtain a P-value for its index of aggregation before subsequent analysis. To compare the significant contribution of each protein, we consider its node degree of connectivity, protein interaction quality for its surrounding interacting partners (including both direct and indirect connected partners through iterations), the protein's significance in both unfiltered global network and phenotype-specific network, and other network characteristics. Our tool also provides the gene/protein PubMed reference citation count for the specific phenotype to help users evaluate the ranked proteins. A family-wise adjusted P-value of all significant ranks against randomized topology-preserving networks are also provided to help assess the rank.
					</p>
			<h4><a href="http://discovery.informatics.iupui.edu/SPINNER/">GEMINE</a></h4>	
					<p>
						GEMINE stands for Gene Expression Mutation Interaction Neighborhood Exploration. This database is developed to store the gene significance data in cancers based on the gene expression and mutations. The raw data has been taken from COSMIC and TCGA and the differentially expressed genes have been filtered and prioritized on the basis of their SPINNER rank. We also calculate the similarity between the genes on the basis of the gene expression data from TCGA and mutation data from COSMIC. We also developed the web application, which is used to browse through the database using PHP and javascript.
					</p>
			<h4><a href="https://play.google.com/store/apps/details?id=com.amaze.QuizActivity&hl=en">Vocab World</a></h4>	
					<p>
						"VOCAB WORLD" is an English Vocabulary Android Application published on Google Play. This application consists of a database of a large pool of words. The application consists of Antonym and Synonym quiz which consists of 10 multiple choice questions having a time limit of 2 minutes. Based on the correct answers the final score is displayed which is saved in the history. The option to review one’s performance is also there with correct answers highlighted green and the wrong answers highlighted red. The review section also consists of the word’s meaning.
					</p>
			<h4><a href="https://github.com/ashishjain1988/GBEER_Analysis_Pipeline/wiki">GBEER Analysis Pipeline</a></h4>	
					<p>
						This is a semi-automated pipeline to run the GBEER tool. GBEER tool is being developed by Friedberg lab which is used to quantify and visualize the evolutionary changes that occur in gene blocks.
					</p>
		
		</div>
	</div>
	<script>
$(document).ready(function() {
	$('#softButton').addClass('active');
});
</script> 