<!DOCTYPE html>
<html>
    <head>
        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
        <title>Ashish Jain</title>
		<link rel='icon' href='../assets/images/ajicon.png'/>
        <link rel="icon" type="image/png" href="../assets/images/icon.jpg">
		<link type='text/css' rel='stylesheet' href='../resources/css/bootstrap-theme.css' />
        <link type="text/css" rel="stylesheet" href='../resources/css/bootstrap-theme.min.css' />
        <link type="text/css" rel="stylesheet" href='../resources/css/bootstrap.css' />
        <link type="text/css" rel="stylesheet" href='../resources/css/bootstrap.min.css' />
		<link rel="stylesheet" href="../assets/css/style.css" type="text/css" media="screen"/>
		<link rel="stylesheet" href="../assets/css/style-website.css" type="text/css" media="screen"/>
        <link rel="stylesheet" href="../assets/css/jquery.dataTables.css" type="text/css" media="screen"/>

		<!-- Custom Fonts -->
		<link href="../resources/font-awesome/css/font-awesome.min.css" rel="stylesheet" type="text/css">
    	<link href="https://fonts.googleapis.com/css?family=Montserrat:400,700" rel="stylesheet" type="text/css">
    	<link href="https://fonts.googleapis.com/css?family=Lato:400,700,400italic,700italic" rel="stylesheet" type="text/css">
    	
    	<!-- Theme CSS -->
        <link type="text/css" rel="stylesheet" href='../assets/css/freelancer.min.css' />
        
        <!-- <script src="//ajax.googleapis.com/ajax/libs/jquery/1.8/jquery.min.js"></script> -->
        <script src="../assets/js/jquery-1.11.1.min.js"></script>
        <script src="../assets/js/jquery.dataTables.min.js"></script>
		<!-- <script src="../assets/js/custom-js.js"></script> -->
        <script src="../resources/js/bootstrap.js"></script>
        <script src="../resources/js/bootstrap.min.js"></script>
       <!--  <script src="../resources/js/spinner-application.js"></script> -->

        <style type="text/css">
           body {
              height: 100%;
              padding: 0;
              margin: 0;
           }

           #app {
              min-height: 100% !important;
              height:100%;
  width:100%;
  overflow:hidden;
              
           }
			.width{
			width: 80%;
			}
			.aParent div {
  float: left;
  clear: none; 
}
           /* #footer {
              position: absolute;
              bottom: 0;
              background: red;
              height: 100px;
              width: 100%;
           } */

        </style>
    </head>
    <body>
    <!-- <div id="app"> -->
        <nav id="mainNav" class="navbar navbar-default navbar-fixed-top navbar-custom">
        <div class="">
            <!-- Brand and toggle get grouped for better mobile display -->
            <div class="navbar-header page-scroll">
                <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
                    <span class="sr-only">Toggle navigation</span> Menu <i class="fa fa-bars"></i>
                </button>
            </div>

            <!-- Collect the nav links, forms, and other content for toggling -->
           <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
                <ul class="nav navbar-nav navbar-right" id="navigation">
				<li id="homeButton" class="page-scroll">
					<a href="../">Home</a>
				</li>
				<li id="researchButton" class="page-scroll">
					<a href="research.html">Research Projects</a>
				</li>
				<li id="publicationsButton" class="page-scroll">
					<a href="publications.html">Publications</a>
				</li>
				<li id="expButton" class="page-scroll">
					<a href="experience.html">Experience</a>
				</li>
				<li id="softButton" class="page-scroll">
					<a href="softwares.html">Softwares</a>
				</li>
				<li id="blogButton" class="page-scroll">
					<a href="skills.html">Skills</a>
				</li>
				<!-- <li id="photoButton" class="page-scroll">
					<a href="../index.php/photography">Photography</a>
				</li> -->
				<li id="contactButton" class="page-scroll">
					<a href="contact.html">Contact</a>
				</li>
                </ul>
            </div>
            <!-- /.navbar-collapse -->
        </div>
        <!-- /.container-fluid -->
    </nav>

<div id="contents" style="margin-top:6%;">
		<div id="about" class="body">
			<h1>Research Projects</h1>
			
			<h4>Bisphenol A and Bisphenol S Disruptions of the Mouse Placenta and Potential Effects on the Placenta-Brain Axis</h4>
			<p>
				Placental trophoblast cells are potentially at risk from circulating endocrine disrupting chemicals, such as bisphenol A (BPA). To understand how BPA and the reputedly more inert bisphenol S (BPS) affect the placenta, C57BL6J mouse dams were fed 200 g/kg body weight BPA or BPS daily for two weeks and then bred. They continued to receive these chemicals until embryonic day 12.5, whereupon placental samples were collected and compared to unexposed controls. BPA and BPS altered the expression of an identical set of 13 genes. Both exposures led to a decrease in the area occupied by spongiotrophoblast relative to multinucleated giant cells (GC) within the junctional zone, markedly reduced placental serotonin (5-HT) concentrations, and lowered 5-HT GC immunoreactivity. Concentrations of dopamine and 5-hydroxyindoleacetic acid), the main metabolite of serotonin, were increased. GC Dopamine-immunoreactivity was increased in BPA and BPS exposed placentas. A strong positive correlation between 5-HT + GC cells and reductions in spongiotrophoblast to GC area suggest this neurotransmitter is essential for maintaining cells within the junctional zone. In contrast, an inverse correlation existed between dopamine + GC cells and ratio of these placental regions. These outcomes lead to the following conclusions: first, BPS exposure causes almost identical placental effects as BPA; second, a major target of BPA/BPS is either spongiotrophoblast or GC within the junctional zone; third, imbalances in neurotransmitter-positive GC and an observed decrease in docosahexaenoic acid and estradiol, also occurring in response to BPA/BPS exposure, likely affect the placental-brain axis of the developing mouse fetus.
			</p>
			
			<h4>Early onset preeclampsia in a model for human placental trophoblast</h4>
			<p>
				We describe a model for early onset preeclampsia (EOPE) that uses induced pluripotent stem cells (iPSC) generated from umbilical cords of EOPE and control (CTL) pregnancies. These were converted to trophoblast (TB) and tested for parameters thought to be disturbed in EOPE, including invasive potential. Under 5 % O2, CTL-TB and EOPE-TB lines showed no differences, but, under 20 % O2, invasiveness of EOPE-TB was reduced (P = 0.008). RNAseq analysis revealed only two differentially expressed genes (RPS17, FDR = 0.0005; MTRNR2L2, FDR = 0.005) significantly down-regulated in EOPE-TB under 20 % O2. A weighted correlation network analysis revealed two gene modules in CTL-TB significantly correlated with both TB invasion and O2 responsiveness (Module CTL4 and CTL9). Out of those two gene modules, CTL9 was positively correlated with 20 % O2 and negatively correlated with TB invasion and was enriched in ontology terms related to “cell migration”, “angiogenesis”, and “pre-eclampsia”. Two EOPE-TB modules EOPE1 and EOPE2 correlated with O2 conditions, but only weakly with invasion; they largely contains the same sets of genes present in CTL modules CTL4 and CTL9. Our experiments suggest that in EOPE, the initial step precipitating disease is a reduced capacity of placental TB to invade, precipitated by a dysregulation of O2 response mechanisms.
			</p>
			<h4>TissueEnrich: A tool to calculate tissue-specific gene enrichment</h4>
			<p>
				Analysis of RNA-Sequencing data results in lists of genes that may have similar function, based on differential gene expression analysis or co-expression network analysis. Multiple tools have been developed that use gene ontologies to identify biological processes that are enriched in the genes sets. While these tools provide insights into the biological processes, there is no information about the tissue specificity of the genes, which is important when studying human disease. Therefore, we developed TissueEnrich, a tool that calculates tissue-specific gene enrichment in an input gene set. We demonstrated that TissueEnrich is very robust in identifying the lineage of single cell clusters and differentiated embryonic stem cells. TissueEnrich is available as a user-friendly and interactive web application as well as an R package allowing additional flexibility in usage.
			</p>
			
			<h4>Deciphering transcriptional regulation in human embryonic stem cells specified towards a trophoblast fate</h4>
			<p>
				Differentiated human embryonic stem cells (hESC) continue to provide a model for studying early trophoblast cells (TB), but many questions have been raised regarding their true identity. Therefore, we carried out a global and unbiased analysis on previously published transcriptomic profiles for hESC differentiated to TB by means of bone morphogenetic protein-4 and inhibitors of activin A and fibroblast growth factor-2 signaling (BAP treatment). Our results confirm that BAP treated hESC (ESCd) lack a mesoderm signature and are a subtype of placental cells unlike those present at term. ESCd display a high level of expression of genes implicated in migration and invasion compared to commonly used, immortalized TB cell lines and primary cells from term placenta. Co-expression network analysis also identified gene modules involved in cell migration and adhesion, processes that are likely critical during the beginning stages of placentation. Finally, protein-protein interaction analysis predicted several additional genes that may play important roles in early stages of placental development. Together, our analyses provide novel insights into the transcriptional programs that are active in ESCd.
			</p>
			
			<h4>Placenta-SeqDB: Curation and annotation of publicly available placenta sequencing dataset</h4>
			<p>
				During the last few years, placental development research has started gaining a lot of attention from the research community. Improvement in NGS techniques has led to an increase in the number of sequencing samples submitted to public databases. GEO is one of the largest data repositories used for storing publically available experimental datasets. However, there are no strict guidelines for submitting the metadata information related to the datasets. There are many discrepancies in the metadata annotation including cell/tissue type, experiment type, and gestational age which make it very difficult to search for specific datasets. Arrayexpress is another such data repository which has a good search engine but it lacks well-annotated placenta-specific terms and features. Because of these issues, searching and retrieving data from these databases poses a great challenge for public data reuse. There are few other databases which have good annotations but don’t have placental datasets. To address this problem, we built a data repository by combining manually curated sequencing datasets including RNA-Seq, ChIP-Seq, and DNAse-Seq from different data repositories including GEO, Arrayexpress, ENA, and CistromeMap. The datasets are linked to the dataset specific genes and diseases through manual curation from research papers. We also developed a new controlled vocabulary based on the curated sample metadata including cell/tissue type, sequencing platform, associated genes, disease, and developmental time points. This resource will help the researchers to search placenta datasets in a quick and efficient manner.
			</p>
			
			<h4>SPINNER</h4>
			<p>
				SPINNER stands for Seeded Protein Interaction Network Neighborhood Expansion and Ranking Algorithm tool. This is an automated tool, which expands the initial seed proteins related to a particular disease, by using the network neighborhood expansion and create protein interaction network for protein network analysis. It expands the protein network by using the Protein-Protein Interaction (PPI) data from the databases like STRING and HAPPI. It also uses a heuristic scoring function for calculating the initial rank score and modified page rank algorithm for calculating the iterative rank score of the proteins in the protein network. This tool also calculates the effect on the network, after the removal of protein from the sub-network. This is called as protein perturbation. As we know, there are certain hub nodes in the protein networks, which play a key role in the activation of the disease. This feature will help us to identify those key hubs which would help in identifying potential drug targets. Using this tool I carried out the analysis of the Amyotrophic lateral sclerosis (ALS) and Breast Cancer.
			</p>
			<!-- <h4>GEMINE (Gene Expression Mutation Interaction Neighborhood Exploration)</h4>
			<p>
				Developed the database which stores the data regarding the significance of genes in cancers based on their gene expression and mutations. The raw data has been taken from COSMIC and TCGA and the differentially expressed genes have been filtered and prioritized on the basis of their SPINNER rank. We also calculate the similarity between the genes on the basis of the gene expression data from TCGA and mutation data from COSMIC. We also developed the web application, which is used to browse through the database using PHP and javascript.
			</p>-->
			<!-- <h4>Smart Health</h4>
			<p>
				Smart Health is an Electronic Medical Record (EMR) Project which was developed in collaboration with Wenzhou Medical Centre, China. The project involves efficient storage of health examination data that has been provided by Wenzhou Medical Centre. We designed and developed the data model and stored the health examination data in an Oracle database. In this process, we developed automated tools to carry out preprocessing and normalization of the data. We also developed a scoring scheme for scoring individuals on the basis of their test results. The scoring scheme can be used to develop prediction models which can be used to predict the potential disease threat on the basis of the current health conditions of an individual.
			</p> -->
			<!-- <h4>Survival Analysis of LGR5 and Co-expressed genes for the discovery of the potential Biomarkers in Colon Adenocarcinoma (Colon Cancer)</h4>
			<p>
			</p> -->
		</div>
	</div>
	<!-- <div class="" style="width:100%; bottom:0px; height: 155px;"> -->
<footer class="text-center">
        <div class="footer-above">
            <div class="container">
                <div class="row">
                <!-- <hr class="star-light"> -->
                    <div class="footer-col col-md-4">
<!--                         <h3>Location</h3> -->
<!--                         <p>2156,Molecular Biology Building -->
<!--                             <br>Iowa State University, Ames, IA</p> -->
                    </div>
                    <div class="footer-col col-md-4">
                         <h3>Around the Web</h3>
                        <ul class="list-inline">
                            <li>
                                <a href="https://www.linkedin.com/in/ashish-jain-395b2711" class="btn-social btn-outline"><i class="fa fa-fw fa-linkedin"></i></a>
                            </li>
<!--                             <li> -->
<!--                                 <a href="#" class="btn-social btn-outline"><i class="fa fa-fw fa-dribbble"></i></a> -->
<!--                             </li> -->
                            <li>
                                <a href="https://github.com/ashishjain1988" class="btn-social btn-outline"><i class="fa fa-fw fa-github-square"></i></a>
                            </li>
                            <li>
                                <a href="https://www.facebook.com/ashish.jain22" class="btn-social btn-outline"><i class="fa fa-fw fa-facebook-f"></i></a>
                            </li>
                            <li>
                                <a href="https://twitter.com/jainashish88" class="btn-social btn-outline"><i class="fa fa-fw fa-twitter-square"></i></a>
                            </li>
                        </ul>
                    </div>
<!--                     <div class="footer-col col-md-4"> -->
<!--                         <h3>About Us</h3> -->
<!--                         <p></p> -->
<!--                     </div> -->
                </div>
            </div>
        </div>
        <div class="footer-below">
            <div class="container">
                <div class="row">
                    <div class="col-lg-12">
                        Ashish Jain Website 2016-17
                    </div>
                </div>
            </div>
        </div>
    </footer>
</body>
</html>
	
	<script>
$(document).ready(function() {
	$('#researchButton').addClass('active');
});
</script> 
